docs/website/docs/plus/getting-started/installation.md

```diff
@@ -3,12 +3,156 @@ title: Installation
 description: Installation information for dlt+
 ---
 
-# Installing
+# Installation
 
-We distribute packages via PyPI, for example to install `dlt-plus` package:
+dlt+ requires a valid license to run, which you can obtain from dltHub by [joining our waiting list](https://info.dlthub.com/waiting-list).
+
+:::info Supported Python versions
+
+dlt+ currently supports Python versions 3.9-3.12.
+
+:::
+
+## Quickstart
+
+To install the `dlt-plus` package, run:
 
 ```sh
 pip install dlt-plus
 ```
 
-Please install a valid license before proceeding, as described below.
\ No newline at end of file
+Please install a valid license before proceeding, as described under [licensing](#licensing).
+
+## Setting up your environment
+
+### Configuration of the Python environment
+
+Check if your Python (versions 3.9-3.12) environment is configured:
+
+```sh
+python --version
+pip --version
+```
+
+If you have a different Python version installed or are missing pip, follow the instructions below to update your Python version and/or install `pip`.
+
+<Tabs values={[{"label": "Ubuntu", "value": "ubuntu"}, {"label": "macOS", "value": "macos"}, {"label": "Windows", "value": "windows"}]}  groupId="operating-systems" defaultValue="ubuntu">
+  <TabItem value="ubuntu">
+
+You can install Python 3.10 with `apt`.
+
+```sh
+sudo apt update
+sudo apt install python3.10
+sudo apt install python3.10-venv
+```
+
+  </TabItem>
+  <TabItem value="macos">
+
+On macOS, you can use [Homebrew](https://brew.sh) to install Python 3.10.
+
+```sh
+brew update
+brew install python@3.10
+```
+
+  </TabItem>
+  <TabItem value="windows">
+
+After installing [Python 3.10 (64-bit version) for Windows](https://www.python.org/downloads/windows/), you can install `pip`.
+
+```sh
+C:\> pip3 install -U pip
+```
+
+  </TabItem>
+</Tabs>
+
+### Virtual environment 
+
+We recommend working within a [virtual environment](https://docs.python.org/3/library/venv.html) when creating Python projects.
```

Comment:
Should we add instructions for uv? (optional)

Reply:
in future sure but no time now imo